pkg/apis/servicecatalog/v1alpha1/defaults.go

```diff
@@ -18,13 +18,28 @@ package v1alpha1
 
 import (
 	"github.com/satori/go.uuid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"time"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
+func SetDefaults_ServiceBrokerSpec(spec *ServiceBrokerSpec) {
+	isValidRelistBehavior := spec.RelistBehavior == "Duration" ||
```

Comment:
Reference `ServiceBrokerRelistBehaviorDuration` and `ServiceBrokerRelistBehaviorManual` constants instead of inline strings?

Reply:
This code shouldn't be there at all, see below.